### PR TITLE
PADV-509.1: Remove unnecessary files and add custom changes

### PR DESCRIPTION
**Pull request by sergivalero20**
## Ticket

https://agile-jira.pearson.com/browse/PADV-509

## Description

This PR remove unnecessary templates for CertPREP theme on edx-platform.

## Testing:

- Enable comprehensive theming.
- Run the LMS: `make dev.up.lms`.
- Set `pearson-vue-theme` on a site.
- Compile the CertPREP theme.
- Verify that the CertPREP theme is applied correctly on Olive.
- Enable Course Operations.
- Test course operations features.

(For more information enter here: [How to enable themes)](https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html#enable-themes)

This site is involved in Course Operation [(Course Operation features)](https://github.com/Pearson-Advance/course_operations/blob/master/pearson_course_operation/docs/course_operations_features_juniper.md).

## Screenshots

Log In:
Log in

Dashboard:
image

## Reviewers

- [x] @Squirrel18 
- [x] @anfbermudezme

### CODE REVIEW

File: edx-platform/pearson-vue-theme/lms/templates/footer.html [hunk 1]
```diff
@@ -169,4 +57,4 @@
   % for url in footer_css_urls:
     <link rel="stylesheet" type="text/css" href="${url}"></link>
   % endfor
-% endif
+% endif
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/footer.html]: Missing end of file line. @sergivalero20

File: edx-platform/pearson-vue-theme/lms/templates/header/navbar-not-authenticated.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+from six import text_type
+
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from openedx.core.djangoapps.user_authn.toggles import should_redirect_to_authn_microfrontend
+%>
+
+<%
+  mktg_site_enabled = static.get_value('ENABLE_MKTG_SITE', settings.FEATURES.get('ENABLE_MKTG_SITE', False))
+  courses_are_browsable = settings.FEATURES.get('COURSES_ARE_BROWSABLE')
+  allows_login = not configuration_helpers.get_value('DISABLE_LOGIN_BUTTON', False)
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/header/navbar-not-authenticated.html]: Why did you remove enable combined registration? @sergivalero20

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-vue-theme/lms/templates/header/navbar-not-authenticated.html (hunk 2 above)
Comment (sergivalero20) [reply edx-platform/pearson-vue-theme/lms/templates/header/navbar-not-authenticated.html]: @Squirrel18 sorry, I'll add it again.

**Review by Squirrel18 — APPROVED** [review]

**Review by anfbermudezme — APPROVED** [review]
